csm/common/ha/cluster_management/operations_factory.py

```diff
@@ -61,7 +61,8 @@ def get_operation(self, operation: str) -> operations.Operation:
             nodeOperation = operations.NodeStopOperation()
         elif operation == "poweroff":
             nodeOperation = operations.NodePoweroffOperation()
-
+        elif operation == "mark_node_failure":
```

Comment:
read this from const -> MARK_NODE_FAILURE

Reply:
Addressed